### DISCUSSION TITLE
Assets not imported

### DISCUSSION THREAD

**Discussion by Mast3rSensei** [apache/superset#26411] [Q&A / Help]
Hi all,

I'm trying to solve the problem that arises when, whenever a dashboard is imported, if the dashboard was already existent, even by overwriting it the relative charts and datasets linked to that dashboard are not updated as well. I read on the Preset docs that, if you actually want an import to update everything, you need to import an _asset_: this should be achieved by simply exporting the dashboard, unzip the file, change the `type` from "Dashboard" to "assets", compress everything again and do an import through the _assets/import/_ endpoint.

So far so good, but when I try to do that, although I receive a code status 200 in response, nothing actually happens at Superset level. This is the request I send when trying to import the assets.

```
files = {
        'formData': (
            paht/to/file.zip,
            open(path/to/file.zip, 'rb'),
            'application/zip'
        )
    }

headers = {
    'Authorization': 'Bearer XXXXX',
    'Referer': 'http://localhost:8088/api/v1/assets/import/'
}

response = requests.post("http://localhost:8088/api/v1/assets/import/", headers=headers, files=files)
```

What am I doing wrong?

**Comment by Pratyush234** [comment]
Instead of 'formData', you need to use 'bundle' and pass CSRF token in your headers
